policyengine_us/parameters/gov/aca/state_rating_area_cost.yaml

```
@@ -8,6 +8,7 @@ metadata:
      - range(1, 68) #max rating areas +1
   documentation: For NY and VT below is base adult premium
   label: Second Lowest Cost Silver Plan premiums by rating area
+  uprating: gov.irs.uprating
```

Need this any time adding a functional metadata field to a nested parameter
```suggestion
  uprating: gov.irs.uprating
  propagate_metadata_to_children: true
```